incorporating-accessibility-into-your-dev-process/linting-and-tests/solution/src/ButtonsAndLinks/ButtonsAndLinks.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
```

Comment:
The spacing seems weird in this file, but if that's on purpose, then ignore me :)

Reply:
I see others the same way, so I assume it is purposeful.

Reply:
I think I’m missing something, what’s weird about it?

Everything is indented with eslint.

Reply:
The extra blank lines in between `<>`  and `<h2>` and `<p>` etc. It just looked different than other files, so I wasn't sure if it was intentional.

Reply:
Ah gotcha, thanks for checking!